src/main/resources/i18n/messages.properties

```diff
@@ -248,6 +248,22 @@ EditUmsConfigurationFileManually=Edit UMS configuration file manually
 Empty=Empty
 Enable=Enable
 EnableAvisynthVariableFramerate=Enable AviSynth variable framerate change into a constant framerate (convertfps=true)
+Avisynth2Dto3DSettings=2D to 3D Conversion Settings
+AlgorithmForConverting2Dto3D=Conversion Algorithm:
```

Comment:
A lot of these keys don't match the convention of the file, which is generally to match the first few words of the string. For example this one would be `ConversionAlgorithm`